### PR TITLE
Add signature help support

### PR DESCRIPTION
**Pull request by NipunaRanasinghe**
## Purpose
$subject.

### CODE REVIEW

File: src/main/java/org/wso2/lsp4intellij/editor/EditorEventManager.java [hunk 1]
```diff
@@ -183,6 +186,8 @@
     private EditorMouseListener mouseListener;
     private EditorMouseMotionListener mouseMotionListener;
 
+    public List<String> completionTriggers;
+    public List<String> signatureTriggers;
```
Comment (nixel2007) [inline src/main/java/org/wso2/lsp4intellij/editor/EditorEventManager.java]: should it be List<Character> ?

Comment (NipunaRanasinghe) [reply src/main/java/org/wso2/lsp4intellij/editor/EditorEventManager.java]: lsp4j has defined both completion-triggers and signature-triggers as `List<String>` (Not sure the reason why) so I preserved the type as it is.

File: src/main/java/org/wso2/lsp4intellij/editor/EditorEventManager.java [hunk 2]
```diff
@@ -259,7 +268,9 @@ public DidChangeTextDocumentParams getChangesParams() {
      * @param c The character just typed
      */
     public void characterTyped(char c) {
-        // Todo - Implement
+        if (signatureTriggers.contains(Character.toString(c))) {
```
Comment (nixel2007) [inline src/main/java/org/wso2/lsp4intellij/editor/EditorEventManager.java]: As i can see, this code expects one-symbol string, cause `c` is a char.
@NipunaRanasinghe

Comment (nixel2007) [reply src/main/java/org/wso2/lsp4intellij/editor/EditorEventManager.java]: so... it could be converted to List<Character> i guess

Comment (NipunaRanasinghe) [reply src/main/java/org/wso2/lsp4intellij/editor/EditorEventManager.java]: Yeah that is one option :) We can either convert the List<String> to List<Character> when reading the server options or we can keep it as it is and do the check like this.. Just a matter of where the conversion should be done right? Which one do you prefer?

Comment (nixel2007) [reply src/main/java/org/wso2/lsp4intellij/editor/EditorEventManager.java]: @NipunaRanasinghe please, add ` around your Lists :D

Comment (NipunaRanasinghe) [reply src/main/java/org/wso2/lsp4intellij/editor/EditorEventManager.java]: @nixel2007 oh my bad :smile: done!

**Pull request merged**